nx-X11/programs/Xserver/hw/nxagent/Clipboard.c

```diff
@@ -923,12 +923,18 @@ void nxagentHandleSelectionRequestFromXServer(XEvent *X)
   int i = nxagentFindLastSelectionOwnerIndex(X->xselectionrequest.selection);
   if (i < nxagentMaxSelections)
   {
+#if 0
```

Comment:
I think this code was only triggered in the old because the handling was not correctly implemented. Nowadays it is superflous. However, as I still do not know how to reach here I would like to keep it for a while. May I could add a comment about this.

Reply:
dropped it now